Makefile

```diff
@@ -9,8 +9,8 @@ else
     b64_identity=$(shell echo '${identity}' | base64 -w 0 -)
 endif
 
-ros_ocp_msg='{"request_id": "uuid1234", "b64_identity": "test", "metadata": {"account": "2234", "org_id": "3340851", "source_id": "111", "cluster_uuid": "222", "cluster_alias": "name222"}, "files": ["http://localhost:8888/ros-ocp-usage.csv"]}'
```

Comment:
Should we have both combinations of data? i.e with and without account?

Reply:
I think it is better to have both as a part of unit test case, instead of having both in Makefile which we just use to ease the work in dev env.